pkg/kubectl/cmd/apply_set_last_applied.go

```diff
@@ -185,7 +186,7 @@ func (o *SetLastAppliedOptions) RunSetLastApplied(f cmdutil.Factory, cmd *cobra.
 				return err
 			}
 			helper := resource.NewHelper(client, mapping)
-			patchedObj, err := helper.Patch(o.Namespace, info.Name, types.MergePatchType, patch)
+			patchedObj, err := helper.Patch(o.Namespace, info.Name, types.StrategicMergePatchType, patch)
```

Comment:
I thought apply and all apply subcommands should use Strategic Merge Patch by default, even though it actual makes no difference in `apply set_last_applied`.